CHANGES.rst

```diff
@@ -7,6 +7,17 @@ Bug Fixes
 jump
 ~~~~
 
+- Added setting of number_extended_events for non-multiprocessing
+  mode. This is the value that is put into the header keyword EXTNCRS.
```

Comment:
Needs to have a PR number listed at the end of the entry. That's why the CI check for a change log entry is failing.